src/klein/resource.py

```diff
@@ -1,7 +1,63 @@
-from ._resource import KleinResource, ensure_utf8_bytes
+# -*- test-case-name: klein.test.test_resource.GlobalAppTests -*-
 
+"""
+This module, L{klein.resource}, serves two purposes:
 
-__all__ = (
-    "KleinResource",
-    "ensure_utf8_bytes",
-)
+    - It's the global C{resource()} method on the global L{klein.Klein}
+      application.
+
+    - It's the module where L{KleinResource} is defined.
+"""
+from sys import modules
+from typing import TYPE_CHECKING
+
+from ._app import resource
```

Comment:
`as _resource` is not strictly necessary, but boy would it make sure there is less potentially weird stuff to track down.

Reply:
Sadly if I do that then mypy gets confused :-). Hopefully this will all go away at some point soon if we can have a better place to expose the global klein app...

Reply:
Oh wait no I’m wrong about that. Hmm. Let me see if I can fix it.